src/components/Buildbaron/BuildBaronTable.tsx

```diff
@@ -0,0 +1,96 @@
+import React from "react";
+import styled from "@emotion/styled/macro";
+import Badge from "@leafygreen-ui/badge";
+import { uiColors } from "@leafygreen-ui/palette";
+import { Table } from "antd";
+import { StyledLink } from "components/styles";
+import { BuildBaron, JiraTicket } from "gql/generated/types";
+import { getDateCopy } from "utils/string";
+
+const { blue, gray } = uiColors;
+
+const columns = [
+  {
+    title: "Related tickets from JIRA",
+    render: (text: string, { key, fields }: JiraTicket): JSX.Element => (
+      <div>
+        <JiraTicketRow jiraKey={key} fields={fields} />
+      </div>
+    ),
+  },
+];
+
+export const BuildBaronTable: React.FC<{
+  eventData: BuildBaron;
+}> = ({ eventData }) => {
+  const searchReturnInfo = eventData?.searchReturnInfo;
+  const jiraIssues = searchReturnInfo?.issues;
+
+  return (
+    <Table
```

Comment:
Could you override the table header color in the antd table? It's a slight shade of grey which I don't think goes well with the background, and the design does not have a special background-color on it: https://mongodb.invisionapp.com/share/C2WKQWRWG5Y#/screens/415337714

Reply:
![image](https://user-images.githubusercontent.com/13104717/93261517-c747f100-f770-11ea-8039-553a19a05244.png)